sessions/react-basics/attributes/src/styles.css

```diff
@@ -0,0 +1,48 @@
+:root {
+  /* Color Variables */
+
+  --nemo: #ff4a11;
+  --granite: #252629;
+  --water: #f3f5f9;
+  --foam: #ffffff;
+}
+
+*,
+*::before,
+*::after {
+  box-sizing: border-box;
+}
+
+body {
+  margin: 0;
+  font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, Helvetica,
+    Arial, sans-serif, "Apple Color Emoji", "Segoe UI Emoji", "Segoe UI Symbol";
+}
+
+.article {
+  width: 50%;
+  margin: 2rem auto;
+  padding: 1rem 3rem;
+  background-color: var(--foam);
+  box-shadow: var(--granite) 0px 2px 8px;
+  text-align: center;
```

Comment:
```suggestion
```
I'm going to establish the WDACT foundation.
**W**eb **D**evelopers **A**gainst **C**entered **T**ext. 😂

Reply:
WDACT sounds good to me! I think `text-align: center` is a relic from before I have implemented `flexbox` and forgot to delete it.